mm2src/lp_swap.rs

```diff
@@ -114,6 +114,8 @@ pub use taker_swap::{calc_max_taker_vol, check_balance_for_taker_swap, max_taker
                      TakerSwapPreparedParams, TakerTradePreimage};
 pub use trade_preimage::trade_preimage_rpc;
 
+use super::database::stats_swaps::update_stats_swap_coins_price_to_db;
```

Comment:
I prefer using the full path starting from `crate::` where it's possible. But I'm not sure which style guide is best. If you know an article regarding this topic, please share :)

Reply:
couldn't find any helpful note online..but personally I think using `use super` will be more efficient since it's calling in same modules on like use `use crate`